blazor/datagrid/adaptive-layout.md

```diff
@@ -142,4 +142,53 @@ The following features are only supported in vertical row rendering:
 * Dialog Editing
 * Aggregate
 * Virtual scroll
-* Toolbar
\ No newline at end of file
+* Toolbar
+
+## Rendering an adaptive layout for smaller screens alone
+
+The Grid component has an option to render an adaptive layout based on the device. This can be achieved by specifying the [AdaptiveUIMode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AdaptiveUIMode) property value as **Mobile**, **Desktop**, or **Both**. The default value of the AdaptiveUIMode property is "Both".
```

Comment:
The DataGrid component has an option to restrict the rendering of an adaptive layout based on the device screen size. By default AdapticeUIMode will be Both, so that the same layout is rendered on both larger and smaller screens. If we want to restrict the rendering based on screensize, we can achieve this requirement defining the AdaptiveUIMode as Mobile.